tests/test_pidfile.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -
+#
+# This file is part of gunicorn released under the MIT license.
+# See the NOTICE for more information.
+
+import errno
+import sys
+
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
+
+import gunicorn.pidfile
+
+
+def mock_patch_builtin(name):
```

Comment:
It would, but I didn't want to introduce a requirement on a newer version of mock. I thought it was you who once had me change `assert_not_called` for this reason.